### PR TITLE
Fix list-export and ts error

### PR DESCRIPTION
**Pull request by academo**
* Fixes a typescript error with yargs
* Fixes the `list-exports` command not working with node packages.

### CODE REVIEW

File: src/bin.ts [hunk 1]
```diff
@@ -26,21 +26,21 @@ yargs
...
-      });
-
-      yargs.option('current', {
-        type: 'string',
-        default: null,
-        demandOption: true,
-        describe:
-          'Current package version - a name of an NPM package, a URL to a tar ball or a local path pointing to a package directory or a single file. (In case it is a directory make sure it contains an `index.d.ts` type definition file.)',
-      });
+      return yargs
```
Comment (academo) [inline src/bin.ts:29]: adding return for yargs to report the correct types to TS.

Comment (leventebalogh) [inline src/bin.ts:29]: **Nit:** maybe we can just return straight away
```suggestion
      (yargs) => yargs.option('prev', { 
      //...
```

**Review by leventebalogh — APPROVED** [review]
Nice one 🙌

File: src/bin.ts (hunk 1 above)
Comment (academo) [reply src/bin.ts:29]: I was thinking on this but I like the readability of having brackets.

**Pull request merged**